tensorboard/components/tf_backend/router.ts

```diff
@@ -88,8 +88,8 @@ export function setRouter(router: Router): void {
 
 function createProdPath(pathPrefix: string, path: string,
     ext: string, params?: URLSearchParams): string {
-
-  const url = new URL(`${window.location.origin}/${pathPrefix}${path}`);
+  const url = new URL(`${window.location.origin}`);
```

Comment:
The `URL` wrapper is working pretty indirectly here, apparently by doing some implicit normalization when the properties are set (like adding a leading slash to the path).  Can you add a comment to explain that?